lib/tasks/evm_test_helper.rb

```diff
@@ -1,6 +1,7 @@
 module EvmTestHelper
-  VMDB_EXCLUDED_SPEC_DIRECTORIES = ["migrations"].freeze
+  VMDB_EXCLUDED_SPEC_DIRECTORIES = %w(replication migrations).freeze
```

Comment:
@martinpovolny This line is going to have to go back to just being migrations.  We previously removed the replication suite.